### PR TITLE
Increased spawned planets to three; map content config list refactor

### PR DESCRIPTION
**Pull request by atakiya**
## About The Pull Request
Look at title

## Why It's Good For The Game
We don't run tradership for a while now, and lavaland takes the single planet slot already.
Rounds also run long enough for people to actually exhaust current ruins and planets.

## Changelog
:cl:
balance: Three planets spawn now per map by default
code: map content config lists aren't spaghetti anymore
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
Lavaland is an extra planet spawn coming from the "minetype" variable on map configs. It doesnt use up any slot.

**Comment by atakiya** [comment]
> Lavaland is an extra planet spawn coming from the "minetype" variable on map configs. It doesnt use up any slot.

Oh. I see.

**Comment by CypressBz** [comment]
so change the 3 to a 2 essentially?

The problem I have, and the reason this came up is that we only see half the planets we have because all stations spawn with a lavaland, and 1 other planet. and a minimum of 1 planet is always one of the 4 easy habitable ones. I want to get the other 4 planets out. 

Might also be worth straight up disabling the lava planet however. setting it's weight to 0 should accomplish that, right?

Also I'm working through a planetary ruin pack I'm building, It's my best ruin mapping yet. I'm getting full use of Az's new flora passthrough mapping helper stuff.

**Comment by atakiya** [comment]
> so change the 3 to a 2 essentially?

>
That’s why I drafted it.

File: code/datums/map_config.dm [hunk 1]
```diff
@@ -59,7 +59,7 @@
 	/// Possible water colors of the loaded map
 	var/list/water_color
 
-	var/amount_of_planets_spawned = 1
+	var/amount_of_planets_spawned = 3
```
Comment (atakiya) [inline code/datums/map_config.dm:62]:
```suggestion
	var/amount_of_planets_spawned = 2
```

**Review by MissFox0810 — APPROVED** [review]

**Review by Azarak — APPROVED** [review]

**Pull request merged**